src/debugger/sourceMap.ts

```diff
@@ -15,7 +15,7 @@ interface ISourceMap {
 }
 
 export class SourceMapUtil {
-    private static SourceMapURLRegex: RegExp = /\/\/(#|@) sourceMappingURL=(.+?)\s*$/m;
+    private static SourceMapURLRegex: RegExp = /\/\/(#|@) sourceMappingURL=(.+index\.ios\.map.*?)\s*$/m;
```

Comment:
What's the advantage of that implementation over the current implementation?

If I remember correctly, in the Movies example app, this file is called Movies.ios.map, so that change will break that example.